apps/greencheck/tests/test_api_asn.py

```diff
@@ -195,12 +232,13 @@ def test_can_only_destroy_asns_for_own_hosting_provider(
         An user must be associated with a hosting provider to be able
          to create or destroy ASNs or IP Ranges for the provider
         """
+        green_asn.hostingprovider = hosting_provider_with_sample_user
         green_asn.save()
         assert gc_models.GreencheckASN.objects.filter(asn=12345).count() == 1
 
         rf = APIRequestFactory()
         url_path = reverse("asn-detail", kwargs={"pk": green_asn.id})
-        hosting_provider_with_sample_user.user_set.first()
+        # hosting_provider_with_sample_user.users.first()
```

Comment:
It doesn't look like this code is doing anything anymore (if it ever did). Would you mind deleting it if so?

Reply:
Good catch!